awx/main/models/credential/__init__.py

```diff
@@ -1092,14 +1092,8 @@ def create(self):
                 'type': 'string',
             },
             {
-                'id': 'password',
-                'label': ugettext_noop('Password'),
-                'type': 'string',
-                'secret': True,
-            },
-            {
-                'id': 'token',
-                'label': ugettext_noop('Access Token'),
+                'id': 'password/token',
+                'label': ugettext_noop('Password/Token'),
```

Comment:
This might break AWX 18.0.0 -> 18.next upgrades if anybody made a credential in the old format (`inputs.password` or `inputs.token` set).  That might be fine, though, considering this functionality doesn't actually have any effect yet.

Also, I don't think that `id` is allowed to have `/` in it; I think it has to be a valid Python identifier.

Reply:
For backwards compatibility, maybe we should just pick one of the old keys e.g., `token`.

Reply:
Hmmm. Does this not require a migration?

Reply:
I arbitrarily chose token, but will change it if need be. Since I combined the two fields into one it may require a migration even if I do just choose an existing id string

Reply:
@shanemcd I'm pretty sure that the definitions for credential types no longer reside in the database, so this won't actually require a data migration.

That said, if somebody sets up a credential w/ the *old* schema and sets a `password` value, it'll just never be used again after this code change.

Reply:
Agreed. Hopefully the number of people who have done this already is fairly minimal.

Reply:
double agreed ^^